miio/integrations/yeelight/__init__.py

```diff
@@ -363,7 +363,10 @@ def set_brightness(self, level, transition=0):
     )
     def set_color_temp(self, level, transition=500):
         """Set color temp in kelvin."""
-        if level > 6500 or level < 1700:
+        color_temp = self._model_info.lamps[
+            YeelightSubLightType.Main
+        ].color_temp  # for now only main light support
```

Comment:
As this doesn't change after the initialization, it'd be cleaner to initialize this already inside `__init__`?
```
self._main_light = self._model_info.lamps[..]
self._color_temp_range = self._main_light.color_temp
```

Reply:
@rytilahti Please look I have added light_type as constructor property and naw I can add something like `miiocli yeelight Background --ip 0.0.0.0 --token ======= set_color_temp 4000` in the next PR. What do you think?

Reply:
The code looks good to me, adding an option of sorts can be discussed later on :)